### DISCUSSION TITLE
how do i rebind bind commands?

### DISCUSSION THREAD

**Discussion by ADAPTATIONOFLONGARMS** [winstxnhdw/lc-hax#407] [Q&A] [answered]
for example, how can i change toggle phantom bind from = over to 0?

**Comment by winstxnhdw** [comment]
https://github.com/winstxnhdw/lc-hax/blob/main/lc-hax/Scripts/Core/InputListener.cs#L35 Change this line to the key you want.

**Reply by ADAPTATIONOFLONGARMS** [reply]
thank you man.

**Reply by ADAPTATIONOFLONGARMS** [reply]
darn it, error happening

**Reply by winstxnhdw** [reply]
What's the error?

**Reply by winstxnhdw** [reply, marked as answer]
`0` would be `(() => Keyboard.current[Key.Digit0].wasPressedThisFrame, () => InputListener.OnEqualsPress?.Invoke()),` right?

**Reply by ADAPTATIONOFLONGARMS** [reply]
oh thats the issue
